gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.45.1
+version=29.46.0
```

Comment:
can you mention how many consumer have you tesetd in the test description? Now I only see "Tested the build process of Java 8, Java 11 and Java 17 consumers against the local snapshot of RestLi."

Reply:
Added.